auth0_flutter/README.md

```diff
@@ -395,11 +394,7 @@ The Credentials Manager utility allows you to securely store and retrieve the us
 When the users open your app, check for valid credentials. If they exist, you can retrieve them and redirect the users to the app's main flow without any additional login steps.
 
 ```dart
-final isLoggedIn = await auth0
-        .webAuthentication()
-        .credentialsManager
-        ?.hasValidCredentials() ??
-    false;
+final isLoggedIn = await auth0.credentialsManager.hasValidCredentials();
```

Comment:
I do want to add that, as a developer, seeing code like this I would expect `auth0.api.login` to result in `hasValidCredentials` to be `true`, which is not true.

Reply:
Thanks for the observations. We currently are referencing that in this README, in the following instances:
- Documentation of `auth0.api.login()`: snippet includes call to `auth0.credentialsManager.storeCredentials()` afterward. https://github.com/auth0/auth0-flutter/pull/132/files#diff-300bb421482a8868624c82300a9e4332736d0cd6684c32fccf6ccd2e60bafa3eR483
- Note in the **Credentials** section: https://github.com/auth0/auth0-flutter/blob/867948bea55804940704656e29794c729eeec9ab/auth0_flutter/README.md#credentials-manager
- Documentation on custom implementations: https://github.com/auth0/auth0-flutter/pull/132/files#diff-300bb421482a8868624c82300a9e4332736d0cd6684c32fccf6ccd2e60bafa3eR418

What else do you suggest we can do to better convey this?